langwrappers/src/main/kotlin/org/jetbrains/research/testspark/langwrappers/PsiComponents.kt

```diff
@@ -90,6 +90,14 @@ interface PsiClassWrapper {
 interface PsiHelper {
     val language: SupportedLanguage
 
+    /**
+     * Checks if a class is available for generation at the given caret offset.
+     *
+     * @param e The AnActionEvent representing the current action event.
+     * @return `true` if a class is available for generation at the caret offset, `false` otherwise.
```

Comment:
`if a class is available ..` -> `if a code construct is valid for unit test generation`.

Reply:
updated